pizzas.html

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+styles:
+  - /css/type.css
+  - /css/pizzas.css
+---
+
+<script src="https://cdn.jsdelivr.net/npm/chart.js"></script> <!-- Chart.js CDN included-->
+
+<div id="pizzaMainContent">
+  <h1>CSS Pizza Statistics</h2>
+
+  {% for years in site.data.pizza %}
+    {% assign year = years[1] %}
+    {% if year.current %}
+      {% assign current_year = year %}
+    {% endif %}
+  {% endfor %}
+  {% assign total_pizzas = 0 %}
+  {% for years in site.data.pizza %}
+    {% if years[1].current %} 
+      {% assign year = years[1] %} 
+      {% for flavour in year.statistics %} 
+        {% assign total_pizzas = total_pizzas | plus: flavour[1] %}
+      {% endfor %}
+    {% endif %} 
+  {% endfor %}
+  <h3>In {{ current_year.academic_year }}, CSS has so far eaten {{ total_pizzas
+  }} pizzas!</h3>
+  
+  <div id="pizzaCharts">
+    <!-- scripting for current year statistics -->
+    {% assign current_labels = "" | split: ',' %}
+    {% assign current_figures = "" | split: ',' %}
+    {% for flavour in current_year.statistics %}
+      {% assign current_labels = current_labels | push: flavour[0] %}
+      {% assign current_labels = current_labels | push: ", " %}
+      {% assign current_figures = current_figures | push: flavour[1] %}
+      {% assign current_figures = current_figures | push: ", " %}
+    {% endfor %}
+   
+    <div id="annualBarChartHolder">
+      <canvas id="annualBarChart"></canvas>
```

Comment:
It is worth adding `aria-label` and `role` attributes here (or something similar) for accessibility. See: https://www.chartjs.org/docs/latest/general/accessibility.html

For example `role="img" aria-label="Vegan Vegi Supreme: X, etc."`, or just saying it's a bar chart instead.